rskj-core/src/main/java/co/rsk/peg/StateForFederator.java

```diff
@@ -53,15 +52,13 @@ public SortedMap<Keccak256, BtcTransaction> getRskTxsWaitingForSignatures() {
      * @return The RLP-encoded byte array representing the current state.
      */
     public byte[] encodeToRlp() {
-        byte[] serializedRskTxsWaitingForSignatures = 
-            BridgeSerializationUtils.serializeRskTxsWaitingForSignatures(rskTxsWaitingForSignatures);
-        return RLP.encodeList(serializedRskTxsWaitingForSignatures);
+        return BridgeSerializationUtils.serializeRskTxsWaitingForSignatures(
```

Comment:
Did you validate this? Sure it's not breaking consensus?

Reply:
Up to my knowledge this shouldn't break consensus since this change affects a read-only method of the Bridge smart contract. I guess if during node sync time this forks we would know if this truly breaks consensus. Maybe we can try syncing a node with this change?

Reply:
maybe a test? :)

Reply:
For serialization/deserialization tests you can find it here: https://github.com/rsksmart/rskj/pull/2738

Reply:
ah ok!! seems good to me then

Reply:
I think we should rollback this change, until we are 100% certain this doesn't break consensus. That will require some further analysis

Reply:
Sounds good, rollback done